Fees page (#3927)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support.md

title: "Teacher training funding"
heading: "Fund your teacher training"
description: |-
    Explore funding for teacher training, including tuition fees, student finance, bursaries and scholarships, and support if you’re disabled, a parent or carer or a veteran.
title_paragraph: |-

>     If you're applying for teacher training in England, we can help you understand how much it may cost and what funding and practical support you could be eligible for.

Author: I didn't think it was misleading as I used may to try and caveat that as it may not cost them anything. And we do something similar with the funding and say could be eligible for as salaried courses aren't eligible for student finance. But I can reword if you think it needs to be clearer.

>     If you're applying for teacher training in England, we can help you understand how much it may cost and what funding and practical support you could be eligible for.

Reviewer: Ignore this - this is the category page so what you have written makes sense (think I was reading it as the fees page!)

>     If you're applying for teacher training in England, we can help you understand the difference between courses with fees and courses with salaries, and what funding and practical support you could be eligible for.

Reviewer: ```suggestion
    If you're applying for teacher training in England, we can help you understand the cost of teacher training and what funding and practical support you could be eligible for.
```

I'm not sure we are helping people to understand the overall differences between courses with fees and salaries? It's also quite a long para. However, I'm not sure my alternative works as it mentions cost!
